modules/planning/open_space/trajectory_smoother/distance_approach_ipopt_interface.cc

```diff
@@ -1379,6 +1379,11 @@ void DistanceApproachIPOPTInterface::finalize_solution(
   free(cind_L);
   free(hessval);
 
+  // debug only
```

Comment:
why not check constant before allocating state/time results?

Reply:
> why not check constant before allocating state/time results?

it is fixed.